game/templates/game/board.html

```
@@ -40,6 +40,24 @@
 
 <body>
 
+    <!-- MOBILE FAB -->
+<div class="m-fab" id="mFab">
+  <button class="m-fab-btn" title="Pause"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)